package.json

```diff
@@ -156,6 +156,7 @@
 	],
 	"main": "./out/extension.js",
 	"dependencies": {
+		"esbuild-linux-64": "^0.12.15",
```

Comment:
Interesting, it works on my system without this dep and I use 64bit Manjaro Linux.

Anyways, it should be listed under devDependencies `npm i --save-dev` as it's only required when developing the extension.

Reply:
accidently commited